packages/icons/icons.json

```diff
@@ -2,3899 +2,3899 @@
+        "tags": "[clipboard, notepad, notebook, copy, paste, transfer, storage, note]",
         "group": "action",
         "codepoint": 61783
     },
     {
         "displayName": "Selection",
         "iconName": "selection",
-        "tags": "selection, collection, circle, ring",
+        "tags": "[selection, collection, circle, ring]",
         "group": "action",
         "codepoint": 62117
     },
     {
         "displayName": "Events",
         "iconName": "timeline-events",
-        "tags": "calendar, timeframe, agenda, diary, day, week, month",
+        "tags": "[calendar, timeframe, agenda, diary, day, week, month, schedule]",
         "group": "interface",
         "codepoint": 62195
     },
     {
         "displayName": "Line chart",
         "iconName": "timeline-line-chart",
-        "tags": "graph, line, chart",
+        "tags": "[graph, line, chart, line chart, upward trend, time series, line graph]",
         "group": "data",
         "codepoint": 62196
     },
     {
         "displayName": "Bar chart",
         "iconName": "timeline-bar-chart",
-        "tags": "graph, bar, chart",
+        "tags": "[graph, bar, chart, horizontal histogram, bar chart]",
         "group": "data",
         "codepoint": 62194
     },
     {
         "displayName": "Applications",
         "iconName": "applications",
-        "tags": "application, browser, windows, platforms",
+        "tags": "[application, browser, windows, platforms, pages, duplicate, copy, page, window, website, files]",
         "group": "interface",
         "codepoint": 61721
     },
     {
         "displayName": "Projects",
         "iconName": "projects",
-        "tags": "drawer, sections",
+        "tags": "[drawer, sections, files, open, directory, folder, ebulliant, briefcase, filing]",
         "group": "interface",
         "codepoint": 62080
     },
     {
         "displayName": "Changes",
         "iconName": "changes",
-        "tags": "arrows, direction, switch",
+        "tags": "[arrows, direction, switch, exchange, upload, download, sync]",
         "group": "action",
         "codepoint": 61766
     },
     {
         "displayName": "Notifications",
         "iconName": "notifications",
-        "tags": "notifications, bell, alarm, notice, warning",
+        "tags": "[notifications, bell, alarm, notice, warning, notification, alert]",
         "group": "interface",
         "codepoint": 62046
     },
     {
         "displayName": "Lock",
         "iconName": "lock",
-        "tags": "lock, engage, connect, join, close",
+        "tags": "[lock, engage, connect, join, close]",
         "group": "action",
         "codepoint": 62002
     },
     {
         "displayName": "Unlock",
         "iconName": "unlock",
-        "tags": "unlock, disengage, disconnect, separate, open",
+        "tags": "[unlock, disengage, disconnect, separate, open, unlocked, plaintext, authenticated, open lock, unsafe, editing]",
```

Comment:
I think that association was made because that icon is used to unlock when you want to edit something (like in Mac system preferences). Let me know if you still want me to remove it though.

Reply:
alright, you can keep it